Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -63,6 +63,7 @@ class BrowserLocationView: UIView {
 
     lazy var urlTextField: UITextField = {
         let urlTextField = UITextField()
+        urlTextField.setContentCompressionResistancePriority(0, forAxis: UILayoutConstraintAxis.Horizontal)
```

Comment:
> Don't need this anymore.

That depends on how we want it to look on the 4S.

Without this line:
<img width="592" alt="screenshot 2015-07-31 17 09 29" src="https://cloud.githubusercontent.com/assets/1093266/9019823/9e52c916-37a7-11e5-8f79-5641144041e0.png">

With this line:
<img width="592" alt="screenshot 2015-07-31 17 10 38" src="https://cloud.githubusercontent.com/assets/1093266/9019824/a518c3f4-37a7-11e5-913c-ebb218ee2308.png">

Note that the latter is how we currently look on master. I wouldn't be opposed to the more compressed version, though, given the very limited screen real estate on the 4S...though that's @darrinhenein's call.